src/components/interfaces/MOBILE_API.xml

```diff
@@ -136,7 +136,7 @@
             <description>The value being set is read only</description>
         </element>
     </enum>
-    
+
```

Comment:
All of these types of changes in this file are unneccessary and don't relate to the proposal. Instead, only the one line related to adding the FILE enum should appear.